services/morph_chain/protocol.privnet.yml

```diff
@@ -30,7 +30,7 @@ ApplicationConfiguration:
   RPC:
     Enabled: true
     EnableCORSWorkaround: false
-    MaxGasInvoke: 15
+    MaxGasInvoke: 100
```

Comment:
the last time i tried to change that value in devenv i've met arguments that all our deployments have that (15) value, and `neofs-adm` must be able to work in such envs. what TX could not fit that limit?

Reply:
> Mainnet has 100. Think, we can safely mirror this.

... or not all our deployments have that...

Reply:
@carpawell yep, it discards #207 resolution.